ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -146,84 +67,85 @@ function CVEsTable({ querySearchFilter, isFiltered, pagination }: CVEsTableProps
                     <Th>First discovered</Th>
                 </Tr>
             </Thead>
-            {tableState.type === 'LOADING' && <TbodyLoading colSpan={colSpan} />}
-            {tableState.type === 'ERROR' && (
-                <TbodyError colSpan={colSpan} error={tableState.error} />
-            )}
-            {tableState.type === 'EMPTY' && (
-                <TbodyEmpty
-                    colSpan={colSpan}
-                    message="No Platform CVEs have been reported for your secured clusters"
-                />
-            )}
-            {tableState.type === 'FILTERED_EMPTY' && <TbodyFilteredEmpty colSpan={colSpan} />}
-            {tableState.type === 'COMPLETE' &&
-                tableState.data.map((nodeCve, rowIndex) => {
-                    const {
-                        cve,
-                        nodeCountBySeverity: { critical, important, moderate, low },
-                        distroTuples,
-                        topCVSS,
-                        affectedNodeCount,
-                        firstDiscoveredInSystem,
-                    } = nodeCve;
-                    const isExpanded = expandedRowSet.has(cve);
+            <TbodyUnified
+                tableState={tableState}
+                colSpan={colSpan}
+                emptyProps={{
+                    message: 'No Platform CVEs have been reported for your secured clusters',
```

Comment:
Off the subject of this refactoring: **Platform** uppercase or **platform** lowercase in this context?

Reply:
Oh, but is this copy-paste? **node** CVEs?

Reply:
Whoops, yes this is a copy-paste issue.

I have internalized these as "Platform CVEs" and "Node CVEs", always capitalized, as they are the name of a feature (right or wrong as that may be). I'd be in favor of changing the wording to:
"No CVEs have been reported for your secured clusters"
and 
"No CVEs have been reported for nodes across your secured clusters"
to side-step the capitalization.

Reply:
* Preposition **for** reads fluently.
* Does workload CVEs have a precedent for not found? I could not find just now.
    It seems like **have been reported** is somewhat ambiguous with the meaning that people **report** CVEs?

Reply:
`reported` -> `detected` is probably more accurate.

Workload CVEs doesn't have a precedent, it shows the "Clear filters" verbiage whenever the results are zero, regardless of applied filters.